src/sdk/modules/smartSessions/smartSessions.test.ts

```diff
@@ -609,6 +648,24 @@ describe("modules.smartSessions.uniPolicy", async () => {
 
     expect(isInstalledBefore).toBe(true)
 
+    // Trust the mock attester. 
+    // We're running on a fork of base sepolia, where necessary modules are registered on the registry and mock attestations are done.
+    const trustAttestersHash = await nexusClient.sendTransaction({
```

Comment:
Can we make this a decorator? 

`nexusClient.trustAttestors()`

(preferably with no args) would be nice.

Reply:
It can live in here: 

![image](https://github.com/user-attachments/assets/a1c8c6e7-1a8e-4a4e-84df-bf6fc95461cd)

Reply:
yeah can do. but I prefer to do it in bootstrap calldata when account gets deployed. 
we could do both.

can we merge this PR? will handle it with new ss upgrade from ss devs

Reply:
Yes sure, will do now after I fix conflicts